db/migrate/20190325175848_create_dynamic_field.rb

```diff
@@ -0,0 +1,31 @@
+class CreateDynamicField < ActiveRecord::Migration[5.2]
+  def change
+    create_table :dynamic_fields do |t|
+      t.string  :string_key,    null: false
+      t.string  :display_label, null: false
+      t.string  :field_type,    null: false, default: 'string'
+      t.integer :sort_order,    null: false
+
+      t.boolean :is_facetable,  null: false, default: false
+      t.string  :filter_label
+      t.string  :controlled_vocabulary
+      t.text    :select_options
+      t.text    :additional_data_json
```

Comment:
Do we want to get rid of additional_data_json since we have select_options?

Reply:
I wanted it leave it so that it would make migrating the data easier, and we can remove it later.

Reply:
👍